docs/process-for-changing-ion-schema.md

```diff
@@ -0,0 +1,173 @@
+
+As before, announcements will be added to the Ion home and news pages.
+
+# Internal steps for considering a proposal
+
+*This section is addressed to the Ion Schema ** **maintainers**.* *It is included here in order to be transparent about the approval process.*
+
+## Requesting an RFC
+
+When a community member creates a feature request in `ion-schema`, we (the Ion Schema **maintainers**) will engage with the requester on the GitHub issue. The **maintainers** will come to a consensus on the proposal; if its value proposition is compelling, we will request an RFC to explore the technical details of its implementation in depth. If the proposal does not add value or goes against the design principles of Ion Schema, we will politely decline. However, this does not preclude the **feature requestor** or any other **contributor** from revising and resubmitting their proposal with new ideas or information.
+
+Either way, the Ion schema **maintainers** must respond with a justification for our decision. For a feature that is approved, a simple message about how the feature will be useful, aligns with the design goals, etc. will suffice. For a rejection, the justification should specifically explain why the idea does not add value to Ion Schema or how it goes against the design principles of Ion Schema.
+
+If we are interested in an idea but **feature requestor** does not want to create an RFC, we may decide to take it up ourselves, but we are not required to do so.
+
+## Approving an RFC
+
+The Ion Schema **maintainers** are responsible for implementing and maintaining both the spec and most of its implementations, so we are the biggest stakeholder. The RFC cannot move ahead without the **maintainers**’ agreement.
+We believe that it should be easy to iterate on Ion Schema, and so the most critical feedback that we can receive is whether something we are introducing is a misfeature or could otherwise cause problems later on. Feedback about the omission of desired features (or part of a feature) will also be considered, but we may decide to defer solving it if there are no backwards compatibility issues that the proposed feature would introduce.
+
+After feedback has been gathered from Ion Schema users and subject-matter experts, the Ion Schema **maintainers** will incorporate the feedback and make the final decision on the RFC.
+
+The outcome of this process will be shared on the `ion-schema` GitHub issue, resulting in the PR being merged or closed as appropriate.
+
+## Creating an RFC for a new Ion Schema version
+
+When we should create a new version depends on our subjective judgment of whether it is a good time for a new version. However, we should consider:
+
+* Are we still in the midst of implementing the last Ion Schema version?
+* How many accepted features are waiting to be released in a new version of the ISL spec?
+* Has anyone specifically asked for a new version to be released?
```

Comment:
Maybe there should also be some attempt to quantify the impact of the new change.

Reply:
While I think impact is a good idea, I think that "bandwidth to work on it" is probably the closest proxy we _should_ use for an open source project. The people/corporations who are funding the project (with their time or money) have different opinions about what is impactful—or more precisely, what is impactful enough to take priority over other projects (internal or external) that also need funding.

I think "bandwidth to work on it" implies "impactful enough that someone is willing to invest time/effort into changing it" and is a reasonable enough proxy for subjective, relative impact. What do you think?